contributors/devel/pull-requests.md

```diff
@@ -77,35 +96,255 @@ $ $RELNOTES -man # for details on how to use the tool
+
+If you are in the not a member, or a member comments `@k8s-bot ok to test`, the intermediate tests will run:
+
+1. Automatic tests run
+    1. travis (go build/fmt and generated docs)
+    1. Jenkins GCE e2e (minimal e2e suite on GCE)
+    1. Jenkins unit/integration (unit and local cluster tests)
+1. If tests fail, resolve issues by pushing edits to your PR branch
+1. If the failure is a flake, a member can comment `@k8s-bot [e2e|unit] test this issue: #<flake issue>`
+
+Once the tests pass, all failures are commented as flakes, or the reviewer adds the labels `lgtm` and `ok-to-merge`, the PR enters the final merge queue. The merge queue is needed to make sure no incompatible changes have been introduced by other PRs since the tests were last run on your PR.
+
+1. The PR enters the merge queue ([http://submit-queue.k8s.io](http://submit-queue.k8s.io))
+1. The merge queue triggers a test re-run with the comment `@k8s-bot test this`
+1. If tests fail, resolve issues by pushing edits to your PR branch
+1. If the failure is a flake, a member can comment `@k8s-bot [e2e|unit] test this issue: #<flake issue>`
+1. If tests pass, the merge queue automatically merges the PR
+
+That's the last step. Your PR is now merged.
+
+Here's a graphic showing the merge process described above in flowchart form:
 
 ![PR workflow](pr_workflow.png)
 
-# Other notes
+Include all the stuff from the pull-request-commands.md doc here
+
+## Comment Commands Reference
+
+Refer to the comment commands [in the commands doc](https://github.com/kubernetes/test-infra/blob/master/commands.md).
+
+## Automation
 
-Pull requests that are purely support questions will be closed and
-redirected to [stackoverflow](http://stackoverflow.com/questions/tagged/kubernetes).
-We do this to consolidate help/support questions into a single channel,
-improve efficiency in responding to requests and make FAQs easier
-to find.
+We use a variety of automation to manage pull requests.  This automation is described in detail [in the automation doc](automation.md).
 
-Pull requests older than 2 weeks will be closed.  Exceptions can be made
-for PRs that have active review comments, or that are awaiting other dependent PRs.
-Closed pull requests are easy to recreate, and little work is lost by closing a pull
-request that subsequently needs to be reopened. We want to limit the total number of PRs in flight to:
+## How the e2e Tests Work
+
+The end-to-end tests will post the status results to the PR. If an e2e test fails, `k8s-ci-robot` will comment on the PR with the test history and the comment-command to re-run that test.  e.g.
+
+>
+The magic incantation to run this job again is @k8s-bot unit test this. Please help us cut down flakes by linking to an open flake issue when you hit one in your PR.
+
+## How Final Merging Works
```

Comment:
The information in this section is duplicated above. Only one should remain, or the information should be divided properly.